recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
-{% set version = "0.2.4" %}
+{% set version = "0.2.5" %}
 
 package:
   name: nilearn
   version: {{ version }}
 
 source:
   fn: nilearn-{{ version }}.tar.gz
-  url: https://pypi.python.org/packages/source/n/nilearn/nilearn-{{ version }}.tar.gz
-  md5: 66145792fb2aebf51e221225bd210534
+  url: https://files.pythonhosted.org/packages/source/n/nilearn/nilearn-{{ version }}.tar.gz
```

Comment:
Looks like there has been a change in the pypi.python.org URLs to access the .tar.gz:
https://bitbucket.org/pypa/pypi/issues/438/backwards-compatible-un-hashed-package

Following https://bitbucket.org/pypa/pypi/issues/438/backwards-compatible-un-hashed-package#comment-27606213, I am switching to files.pythonhosted.org.